content/20.pharmaceuticals.md

```diff
@@ -835,7 +838,7 @@ While still nascent, work in this area is promising.
 Over the longer term, this approach and others may lead to the development of novel therapeutics specifically for COVID-19 and SARS-CoV-2.
 <!-- This section could be updated with most recent information on drug discovery efforts -->
 
-#### Conclusions
+#### Conclusions <!--I’m not sure we need this section. We’ve stated these things before in this manuscript and the section reads as how useful it’s been to learn from SARS-CoV-2 and MERS-CoV – it has been useful, but I don’t think we need that as the overall conclusion…-->
```

Comment:
I feel a little weird ending on the potential avenues section, if anyone has any ideas for how to flesh out conclusions, that would be awesome!

Reply:
I think the mini conclusion that is there is sufficient. As Simina says we conclude a lot within the manuscript.

Reply:
So do we want to keep it or no? Either way is fine with me!

Reply:
I always feel weird ending a review without some sort of concluding part, but I will let this one up to you folks :)

Reply:
I think I feel the same as @RLordan -- leave it for now, but maybe we can add the table here to give it more teeth?